websites/S/sekolahmu/presence.ts

```diff
@@ -0,0 +1,89 @@
+// it triggers querySelector and querySelectorAll
+/* eslint-disable ts/no-deprecated */
+import { Assets } from 'premid'
+
+const presence = new Presence({
+  clientId: '1037219986378338305',
+})
+const browsingTimestamp = Math.floor(Date.now() / 1000)
+enum ActivityAssets {
+  Logo = 'https://raw.githubusercontent.com/vintheweirdass/sekolahmu-premid-assets/refs/heads/main/images/logo-v2.png',
+}
+presence.on('UpdateData', () => {
+  // sekolahmu is SSR'ed, so sometimes the previous data can showed up unexpectedly.
+  // we need to make a pause for some milliseconds to prevent that
+  setTimeout(async () => {
+    const presenceData: PresenceData = {
+      largeImageKey: ActivityAssets.Logo,
+      startTimestamp: browsingTimestamp,
+      state: 'Viewing LMS',
+      smallImageKey: Assets.Search,
+    }
+    const { querySelector, querySelectorAll, location } = document
+    const { hostname, pathname, search } = location
```

Comment:
For `queySelector` and `querySelectorAll`, you should use them with `document.querySelector(...)` rather than destructuing them
```suggestion
    const { hostname, pathname, search } = document.location
```